### PR TITLE
fix(dev): fix Localstack yet again

### LINKED ISSUE #202: Submitting attachments on Storage Mode does not work in development environment
Submitting attachments recently broke. Possible reasons include problems with localstack integration, incompatible upgrade of AWS SDK, or some other issue that needs to be investigated.

### PR DESCRIPTION
**Pull request by mantariksh**
## Problem

All Localstack services (image upload, logo upload, storage mode attachments) were not working.

Closes #202 

### Initialising Localstack using `awslocal`
In their latest version, [the `awslocal` CLI](https://github.com/localstack/awscli-local) switched to using the edge port 4566 for all services by default. However, this edge port is not available to us, since we are using an older version of Localstack. Hence it is now necessary to specify the endpoint URL in `init-localstack.sh` when creating the S3 buckets.

### Specifying mock AWS credentials
All actions involving S3 require the global `AWS` object to have its credentials configured. Since we removed the mock AWS credentials, those credentials could no longer be set, resulting in all S3 operations failing. The solution was to restore the mock credentials to `docker-compose`.

### What about upgrading Localstack?
Sadly, the [issue](https://github.com/localstack/localstack/issues/2407) opened by our esteemed @karrui on the Localstack repo doesn't seem to have been resolved, since I tried upgrading Localstack but ran into the same problem. We seem to be stuck with v0.8.0 for now.

### CODE REVIEW

**Comment by karrui** [comment]
OH MY GOD YOU FIXED IT

**Comment by mantariksh** [comment]
> OH MY GOD YOU FIXED IT

This is not the issue that is causing Travis builds to fail though! This is just fixing the dev environment

**Review by arshadali172 — APPROVED** [review]

File: docker-compose.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ services:
       - LOGO_S3_BUCKET=local-logo-bucket
       - FORMSG_SDK_MODE=development
       - BOUNCE_LIFE_SPAN=1800000
+      - AWS_ACCESS_KEY_ID=fakeKey
```
Comment (arshadali172) [inline docker-compose.yml:29]: Can we document these env vars in the readme?

Comment (mantariksh) [reply docker-compose.yml:29]: Haven't these always been in `DEPLOYMENT_SETUP.md` though?

**Review by karrui — APPROVED** [review]

**Pull request merged**